public/app/core/components/SharedPreferences/SharedPreferences.test.tsx

```diff
@@ -193,7 +192,7 @@ describe('SharedPreferences', () => {
       timezone: 'browser',
       weekStart: '',
       theme: '',
-      homeDashboardUID: undefined,
```

Comment:
We're removing the `Default` option from the select, which allows the user to set the default Grafana home page. Is this intentional?

|Default value selected|Home page when Default is selected|
|-|-|
<img width="708" alt="Captura de Pantalla 2022-08-26 a las 10 01 14" src="https://user-images.githubusercontent.com/5699976/186853470-9a38900e-11f9-4ddf-b1a5-9e15b8d21382.png">|<img width="1728" alt="Captura de Pantalla 2022-08-26 a las 10 01 05" src="https://user-images.githubusercontent.com/5699976/186853477-c9c9fb51-3882-442d-b6c9-7ea6d900384f.png">

Reply:
We now use the standard "x" to clear the custom home dashboard and go back to the default one.  I updated the language so it is now:
![2022-08-26_08-42-44 (1)](https://user-images.githubusercontent.com/705951/186943078-4e1c70b5-b784-4d34-a5d7-454fd4eb7629.gif)